dymos/transcriptions/pseudospectral/components/pseudospectral_timeseries_output_comp.py

```diff
@@ -150,35 +150,49 @@ def _add_output_configure(self, name, units, shape, desc='', src=None, rate=Fals
         self._vars[name] = (input_name, output_name, shape, rate)
 
         size = np.prod(shape)
-        jac = np.zeros((output_num_nodes, size, input_num_nodes, size))
 
         if rate:
             mat = self.differentiation_matrix
         else:
             mat = self.interpolation_matrix
 
-        for i in range(size):
-            if _USE_SPARSE:
```

Comment:
The _USE_SPARSE setting was just used to quickly toggle and check performance between sparse and dense implementations.  Let's remove the `_USE_SPARSE` variable and just assume that we always do so.  Any performance benefits of dense are typically only present in smaller problems.